src/pages/EventAdministration/components/EventFormAdmin.tsx

```diff
@@ -25,21 +105,68 @@ const EventFormAdmin = ({ eventId }: EventFormAdminProps) => {
 
   const EventFormEditor = ({ formType }: EventFormEditorProps) => {
     const createForm = useCreateForm();
+    const showSnackbar = useSnackbar();
+    const { data: formtemplates = [] } = useFormTemplates();
+    const [isCreatingForm, setIsCreatingTemplate] = useState(false);
+    const [formtemplateId, setFormtemplateId] = useState('-');
+    const { data: form, isLoading, isError } = useFormById(formtemplateId);
 
-    const newForm: EventFormCreate = {
+    const newForm: FormCreate = {
       title: `${event.title} - ${formType === EventFormType.SURVEY ? 'påmeldingsskjema' : 'evalueringsskjema'}`,
       type: formType,
       event: event.id,
       resource_type: FormResourceType.EVENT_FORM,
       fields: [],
     };
 
+    const newFormTemplateCreate: FormCreate = {
+      title: 'BLANK',
```

Comment:
Kan kanskje finne en bedre default tittel? For eks:
```suggestion
      title: 'Ny mal',
```